ogre/src/OgreGpuRays.cc

```diff
@@ -163,6 +163,9 @@ void OgreGpuRays::Init()
 //////////////////////////////////////////////////
 void OgreGpuRays::Destroy()
 {
+  if (!this->dataPtr->ogreCamera)
```

Comment:
[question] I'm assuming the code kinda guarantees that if `this->dataPtr->ogreCamera` is `nullptr`, then the rest will also be `nullptr`, right?

Reply:
It's a backport changes from external source; same below